packages/batch-submitter/src/batch-submitter/tx-batch-submitter.ts

```diff
@@ -390,11 +389,8 @@ export class TransactionBatchSubmitter extends BatchSubmitter {
     }
 
     let isEqual = true
-    const [
-      queueEleHash,
-      timestamp,
-      blockNumber,
-    ] = await this.chainContract.getQueueElement(queueIndex)
+    const [queueEleHash, timestamp, blockNumber] =
+      await this.chainContract.getQueueElement(queueIndex)
```

Comment:
Hmmm not sure if I *love* this but I also don't particularly care either way.

Reply:
We can change the rule if you'd like, but it should out of scope of this PR. The reason this diff was in the PR was because this file was saved and our current formatting rules (i.e., prettier) dictate this style and this PR turns on "formatOnSave"

Reply:
I see. Do you mind making another PR after this is merged that runs `prettier`?

Reply:
Yes. We'll need to co-ordinate though as many files will be changed. We may want to try to modify the prettier config in such a way to minimize this formatting diff.